NitroxServer-Subnautica/GameLogic/Entities/Spawning/ReefbackSpawnData.cs

```diff
@@ -5,42 +5,30 @@ namespace NitroxServer_Subnautica.GameLogic.Entities.Spawning.EntityBootstrapper
 {
     public static class ReefbackSpawnData
     {
-        public struct ReefbackEntity
+        public struct ReefbackCreature
```

Comment:
Let's refactor into a single struct: `ReefbackEntity`.  This would allow us to simplify the reefback bootstrapper into a single function with two invocations:

```
   spawnEntities(spawnablePlants, plantsProbabilitySum, parentEntity, deterministicBatchGenerator);
   spawnEntities(spawnableCreatures, creatureProbabilitySum, parentEntity, deterministicBatchGenerator);

    private void SpawnPlants(List<ReefbackEntity> possibleEntities, float probabilitySum, Entity parentEntity, DeterministicBatchGenerator deterministicBatchGenerator)
```